[2022-07-19 10:56:13,573] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T10:55:00+00:00 [queued]>
[2022-07-19 10:56:13,618] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T10:55:00+00:00 [queued]>
[2022-07-19 10:56:13,620] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 10:56:13,621] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 10:56:13,623] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 10:56:13,692] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): respond> on 2022-07-19 10:55:00+00:00
[2022-07-19 10:56:13,710] {standard_task_runner.py:52} INFO - Started process 9512 to run task
[2022-07-19 10:56:13,736] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'respond', 'scheduled__2022-07-19T10:55:00+00:00', '--job-id', '487', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpcq6dkgg0', '--error-file', '/tmp/tmp03bi51jd']
[2022-07-19 10:56:13,741] {standard_task_runner.py:80} INFO - Job 487: Subtask respond
[2022-07-19 10:56:14,084] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.respond scheduled__2022-07-19T10:55:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 10:56:14,391] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T10:55:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T10:55:00+00:00
[2022-07-19 10:56:25,638] {logging_mixin.py:115} INFO - (2724, 1)
[2022-07-19 10:56:25,654] {python.py:173} INFO - Done. Returned value was: Greet Responded Again
[2022-07-19 10:56:25,701] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=respond, execution_date=20220719T105500, start_date=20220719T105613, end_date=20220719T105625
[2022-07-19 10:56:25,763] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 10:56:25,826] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
